src/shared/components/Checkbox.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import {observer} from "mobx-react";
+import DefaultTooltip from "./defaultTooltip/DefaultTooltip";
+
+export interface ICheckboxProps {
+    value?:string;
+    checked:boolean;
+    onClick:()=>void;
+    label:any;
+}
+
+@observer
+export default class Checkbox extends React.Component<ICheckboxProps, {}> {
```

Comment:
there is already `react-bootstrap` checkbox. how different is this from that?

Reply:
agreed.  if this is different, name should describe what it is.

Reply:
the difference is the flex alignment, should I change the name to FlexAlignedCheckbox

Reply:
I think even that does it right